bridgetown-website/Gemfile

```diff
@@ -8,7 +8,7 @@ gem "bridgetown-core", path: "../bridgetown-core"
 gem "bridgetown-paginate", path: "../bridgetown-paginate"
 
 group :bridgetown_plugins do
-  gem "bridgetown-feed", "~> 1.0", github: "bridgetownrb/bridgetown-feed", branch: "support-resources"
+  gem "bridgetown-feed", "~> 2.0", github: "bridgetownrb/bridgetown-feed", branch: "support-resources"
```

Comment:
@jaredcwhite I had to upgrade this to get the website to build locally. Is this something we want to do?

Reply:
Yeah I updated it already in the 0.21 beta branch but not main. You can take out the github/branch stuff if you want, I already published the gem.

Reply:
Cool will do 👍